terraform/monitoring/04_slos.tf

```diff
@@ -67,16 +57,16 @@ resource "google_monitoring_slo" "frontend_availability_slo" {
   }
 }
 
-# Create another SLO on the frontend service this time with respect to latency.
-# Our SLO is defined as following:
-#   80% of requests that return 200 OK responses return in under 500 ms
-resource "google_monitoring_slo" "frontend_latency_slo" {
-  service = google_monitoring_custom_service.frontend.service_id
+# Create an SLO with respect to latency using the Istio service.
+# Example SLO is defined as:
+#   99% of requests that return 200 OK responses return in under 500 ms
+resource "google_monitoring_slo" "istio_service_latency_slo" {
+  count = length(var.istio_services)
+  service = "ist:${var.project_id}-zone-${var.zone}-stackdriver-sandbox-default-${var.istio_services[count.index].service_id}"
```

Comment:
cloud-ops-sandbox instead of stackdriver-sandbox

Reply:
Fixed :)